webapi/Controllers/ChatController.cs

```diff
@@ -415,6 +412,28 @@ private async Task RegisterPlannerHostedFunctionsUsedAsync(CopilotChatPlanner pl
         return;
     }
 
+    private static ContextVariables GetContextVariables(Ask ask, IAuthInfo authInfo, string chatId)
+    {
+        const string UserIdKey = "userId";
+        const string UserNameKey = "userName";
+        const string ChatIdKey = "chatId";
+
+        var contextVariables = new ContextVariables(ask.Input);
+        foreach (var variable in ask.Variables)
+        {
+            if (variable.Key != UserIdKey && variable.Key != UserNameKey)
```

Comment:
I know this is copied from somewhere else. But what is the point of this condition? We are going to override the values anyways.

Reply:
Yes, this check is redundant.  I'll remove it.